src/Access/AccessControl.h

```diff
@@ -144,6 +146,9 @@ class AccessControl : public MultipleAccessStorage
         const String & forwarded_address,
         const String & custom_quota_key) const;
 
+    static void setAuthTypeSetting(const bool allow_plaintext_and_no_password){ UsersConfigAccessStorage::setAuthTypeSetting(allow_plaintext_and_no_password);}
+    static bool getAuthTypeSetting() {return UsersConfigAccessStorage::ALLOW_PLAINTEXT_AND_NO_PASSWORD;}
```

Comment:
The name `getAuthTypeSetting` is meaningless. Let's rename it to `isPlaintextPasswordAllowed()` and `isNoPasswordAllowed()`